docs/flow_filtering.md

```diff
@@ -0,0 +1,95 @@
+# eBPF Flow Rule Based Filtering
+
+## Introduction 
+
+Flow rule-base filtering is a method to control the flow of packets cached in eBPF's flows table based on certain configuration
+
+## Flow filter rule configuration
+
+ The Flow filter rule consists of two parts mandatory and optional parameters.
+ 
+### Mandatory parameters
+
+- `FLOW_FILTER_IP_CIDR` - IP address and CIDR mask for the flow filter rule, supports IPv4 and IPv6 address format.
+  If wanted to match against any IP, user can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6 respectively.
```

Comment:
IMO `0.0.0.0/0` or `::/0` is not a good example, where you're accepting everything (current state) or rejecting everything, specific example of specific CIDR/IP address would be helpful.

Reply:
that doesn't mean u are accepting see the fine tune section u can use the different optional configs to narrow it down for example I wanted to see all flows with `sport 80` regardless where it came from or going to